README.md

```diff
@@ -407,6 +429,23 @@ This restricts white label SSO use cases and is a challenge for smaller deployme
 Instead we currently accept the navigational tracking since there is no clear path to removing `window.open` from the platform.
 Whether or not this is acceptable will depend on further analysis and discussion.
 
+### Allowing the relying party to control credentials that appear in the credential chooser
+
+Since any site can claim to be an identity provider with any `"effectiveType"`, we may want to allow websites further control over the elements in the UI.
+However this carries a risk of information leak to the relying party of all of the origins of a given type.
+Currently the relying party may mitigate this by validating the origin of the returned credential, or by attempting to use the credential, and by repeating the authentication process if it is unacceptable.
+Here is an example of such behavior in some abstracted Javascript:
+
+```js
+while (true) {
+  let cred = navigator.credentials.create(options);
```

Comment:
I'm probably misunderstanding, but wouldn't this make more sense if used with `navigator.credentials.get`?

Reply:
whoops, you're right.